ghost/i18n/locales/it/newsletter.json

```diff
@@ -1,24 +1,24 @@
 {
-    "By {authors}": "",
-    "Comment": "",
-    "complimentary": "",
-    "Email": "",
-    "free": "",
-    "Keep reading": "",
-    "Less like this": "",
-    "Manage subscription": "",
-    "Member since": "",
-    "More like this": "",
-    "Name": "",
+    "By {authors}": "di",
+    "Comment": "Commenta",
+    "complimentary": "Omaggio",
+    "Email": "Email",
+    "free": "gratis",
+    "Keep reading": "Continua a leggere",
+    "Less like this": "Non mi piace",
+    "Manage subscription": "Gestisci l’abbonamento",
+    "Member since": "Abbonato dal",
+    "More like this": "Mi piace",
+    "Name": "Nome",
     "paid": "",
-    "Subscription details": "",
-    "trialing": "",
-    "Unsubscribe": "",
-    "View in browser": "",
-    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "",
-    "Your free trial ends on {date}, at which time you will be charged the regular price. You can always cancel before then.": "",
-    "Your subscription has been canceled and will expire on {date}. You can resume your subscription via your account settings.": "",
-    "Your subscription has expired.": "",
-    "Your subscription will expire on {date}.": "",
-    "Your subscription will renew on {date}.": ""
+    "Subscription details": "Dettagli",
+    "trialing": "in prova",
+    "Unsubscribe": "Disiscriviti",
+    "View in browser": "Vedi nel browser",
+    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "Ricevi questa mail perché sei <strong>%%{status}%% abbonato</strong> a {site}.",
```

Comment:
to offer a little more context here, the translations for 'free', trialing, paid (which needs a translation still), and complimentary will be subbed in where status is.  Is this the right position for it in the translation? I think these translations may need some revision?

Reply:
I haven't translated 'paid' because I couldn't find the position of the string in the context.json — could you please show me where these lines are printed? "In prova" and "Gratis" should be fine, but "Paid" might be translated as both "Pagato" or "A pagamento" depending on the actual meaning.

Reply:
Here is where paid may show up:
    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "Ricevi questa mail perché sei <strong>%%{status}%% abbonato</strong> a {site}.",

%%{status}%% will be replaced with the strings for free, paid, complimentary, or trialing, as appropriate for that particular user.  So, if it should be 'abbonato gratis' instead of 'gratis abbonato', then you will need to rearrange a bit. :)

Reply:
Thank you! I've changed the wording from "sei abbonato" ("you are subscribed") to "hai un abbonamento" ("you have a subscription") — it flows better in Italian when the modifier directly relates to the noun.